### DISCUSSION TITLE
Ambient Weather WS-1200 Rain Gauge Values

### DISCUSSION THREAD

**Discussion by charilebrown987** [merbanan/rtl_433#2818] [Q&A] [answered]
I just got my RTL_433 receiver connected and trying to normalize the data for use.  The rain gauge data seems off, reporting over 1000mm which seems odd. See json data from payload below.

{"time":"2024-02-05 14:27:39","model":"Fineoffset-WH24","id":220,"battery_ok":1,"temperature_C":12.3,"humidity":97,"wind_dir_deg":248,"wind_avg_m_s":0,"wind_max_m_s":0,"rain_mm":1086.90002,"uv":233,"uvi":0,"light_lux":7599.0,"mic":"CRC"}

Not sure if that's normal for this weather station.  If this is the correct value then what exactly is this so I know how to convert to some meaningful values, etc Rain Rate in in/h.  Data being sent back from Ambient Weather is correct.

**Comment by zuckschwerdt** [comment, marked as answer]
That value is the accumulated total rain since power up. You have to derive values for hourly, daily, etc. intervals yourself.
